2016-07-14-new-news.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: PETER ARTHUR WINS DUBLIN 5000m TRACK TITLE
```

Comment:
Ok Mike thanks I did a straight cut and paste on title. I remember now you did tell me about lowercase